src/components/ContentSidebar/ActivitySidebar.js

```diff
@@ -30,7 +30,8 @@ type Props = {
     mentionSelectorContacts?: SelectorItems,
     commentsError?: Errors,
     tasksError?: Errors,
-    getAvatarUrl: (string) => Promise<?string>
+    getAvatarUrl: (string) => Promise<?string>,
+    getUserProfileUrl?: Function
```

Comment:
Do you want to type this the same way you typed the other `getUserProfileUrl` props?

Reply:
yup! I missed this one!